### PR TITLE
[v2.6] Fixed intermittent failure with TestCreateNamespaceProjectMember test case

### PR DESCRIPTION
**Pull request by igomez06**
## Issue: 
 
https://github.com/rancher/rancher/issues/39710
 
## Problem
 
There was a bit of a timing issue when adding a `project-member` role to a user in order to create a namespace, where the `project-member` role is not fully set causing a `Resource type [namespace] is not creatable` error. 
## Solution

 I added two extra checks to make sure the ProjectRoleTemplateBinding is successfully created.
## Testing

CI drone tests pass.

### CODE REVIEW

**Review by caliskanugur — DISMISSED** [review]
Looks good to me

**Review by superseb — DISMISSED** [review]

**Comment by igomez06** [comment]
Pushed up one more check, this is to ensure the service account secret is properly created in order for a project member to create a namespace.

File: scripts/test [hunk 1]
```diff
@@ -89,7 +89,7 @@ echo Running: build-token-generator
 path=$(pwd)
 ./tests/v2/integration/bin/integrationsetup
 export CATTLE_TEST_CONFIG=${path}/config.yaml
-go test -v ./tests/v2/integration/...
+go test -timeout 60m -v ./tests/v2/integration/...
```
Comment (superseb) [inline scripts/test:92]: If this runs longer than a few minutes, we know it's broken right? Should we adjust the timeout to a few minutes? Or was there another reason to make it 60?

**Review by superseb — DISMISSED** [review]

File: scripts/test (hunk 1 above)
Comment (igomez06) [reply scripts/test:92]: I forgot to remove this, I tried this out for a debugging. I'll remove it

**Review by superseb — APPROVED** [review]

**Review by caliskanugur — APPROVED** [review]

**Pull request merged**